File: example/src/main/java/com/xwray/groupie/example/item/FullBleedCardItem.java
package com.xwray.groupie.example.item;

import android.support.annotation.ColorRes;

import com.xwray.groupie.example.MainActivity;

public class FullBleedCardItem extends CardItem {

    public FullBleedCardItem(@ColorRes int colorRes) {
        super(colorRes);
        getExtras().put(MainActivity.INSET_TYPE_KEY, MainActivity.INSET);
    }
}

File: example/src/main/java/com/xwray/groupie/example/item/FullBleedCardItem.kt
package com.xwray.groupie.example.item

import androidx.annotation.ColorRes
import com.xwray.groupie.example.INSET_TYPE_KEY

class FullBleedCardItem(@ColorRes colorInt: Int) : CardItem(colorInt) {

    init {
        extras.remove(INSET_TYPE_KEY)
    }
}
